ecl/hqlcpp/hqlsource.cpp

```diff
@@ -7253,6 +7253,12 @@ ABoundActivity * HqlCppTranslator::doBuildActivityFetch(BuildCtx & ctx, IHqlExpr
         OwnedHqlExpr simple = replaceFetchInput(expr, null, no_right);
         OwnedHqlExpr transformed = replaceExpression(simple, tableExpr, projected);
         OwnedHqlExpr optSimple = optimizeHqlExpression(queryErrorProcessor(), transformed, optFlags);
+        IHqlExpression * fetch = queryFetch(optSimple);
```

Comment:
@ghalliday minor: may be better not to have this 'fetch' variable shadow the 'fetch' variable in parent scope.

Reply:
agreed, I will fix.